src/main/resources/application.yml

```diff
@@ -9,7 +9,8 @@
 #server.port 9001 used for development purposes
 #change port to 0 when service can be linked to Zuul
 server:
-  port: 50222
+  #port: 50222

```

Comment:
What was this file changed?

Reply:
this file was changed to connect to our database.  It has been changed back